src/patternfly/components/Menu/menu.scss

```diff
@@ -6,7 +6,7 @@
   --pf-c-menu--BoxShadow: var(--pf-global--BoxShadow--md);
   --pf-c-menu--MinWidth: 100%;
   --pf-c-menu--Width: auto;
-  --pf-c-menu--MaxHeight: auto;
+  --pf-c-menu--MaxHeight: auto; // remove at breaking change
```

Comment:
This component is beta, so you can remove the var if you need to - but looks like it's still being used?